Bottom: 24fa5d0e0055ffe28a549acc4cb9fc0929ae9127
Top:    982c8499342fb50cdc846d95f159f539b18e27d2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-12 22:24:59 +0100

Try something with the rtrees









---

diff --git a/src/polyarea.h b/src/polyarea.h
index 42acc1a..819b074 100644
--- a/src/polyarea.h
+++ b/src/polyarea.h
@@ -128,6 +128,7 @@ struct POLYAREA
 {
     POLYAREA *f, *b;
     PLINE *contours;
+    void *contour_tree;
 };
 
 BOOLp poly_M_Copy0(POLYAREA ** dst, const POLYAREA * srcfst);
